docs/user-guide/styling-your-docs.md

```diff
@@ -37,6 +39,8 @@ A clone of the default theme used by the [Read the Docs] service. This theme
 only supports features in its parent theme and does not support any MkDocs
 [theme configuration options] in addition to the defaults.
 
+The mkdocs theme is located in the mkdocs repository at `mkdocs/themes/readthedocs/`.
```

Comment:
This should be: "The **readthedocs** theme..."

Reply:
Oops, good catch!